ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -429,7 +429,10 @@ objects:
         fi
         if [[ "${CLUSTER_TYPE}" = "aws" ]]; then
             master_type=null
-            if [[ "${CLUSTER_VARIANT}" =~ "xlarge" ]]; then
+            worker_type=m4.xlarge
+            if [[ "${CLUSTER_VARIANT}" =~ "xlarge_worker" ]]; then
```

Comment:
We need to separate worker from masters.  `large` and `xlarge` apply to clusters.  I should be able to set both of these.

Naming wise I would recommend `large_worker` (we don't need an xlarge variant).  Use `m4.2xlarge` please (or m5.2xlarge`).  Because the names overlap you will need to update the regex's to terminate on boundaries.  Should be `(^|,)large($|,)` as the regex (you may want to test that on bash first, escaping might be weird.

Reply:
here's the correct syntax to use:
```
if [[ "${C}" =~ (^|,)xlarge_worker($|,) ]]; then
  echo "xlarge_worker"
fi

if [[ "${C}" =~ (^|,)large_worker($|,) ]]; then
  echo "large_worker"
fi

if [[ "${C}" =~ (^|,)xlarge($|,) ]]; then
  echo "xlarge"
fi
if [[ "${C}" =~ (^|,)large($|,) ]]; then
  echo "large"
fi
```